superbuild/projects_modules/ITK.cmake

```diff
@@ -42,7 +42,7 @@ if (NOT USE_SYSTEM_${ep})
 ## #############################################################################
 
 set(git_url ${GITHUB_PREFIX}InsightSoftwareConsortium/ITK.git)
-set(git_tag v5.0.0)
+set(git_tag origin/release)
```

Comment:
why not v5.1.0 ?

Reply:
It didn't exist at the time. But it should be nice to do indeed.